persistence-api/src/main/scala/com/evolutiongaming/akkaeffect/persistence/Snapshot.scala

```diff
@@ -0,0 +1,21 @@
+package com.evolutiongaming.akkaeffect.persistence
+
+import java.time.Instant
+
+trait Snapshot[S] {
+
+  def snapshot: S
+  def metadata: Snapshot.Metadata
+
+}
+
+object Snapshot {
+
+  final case class Metadata(seqNr: SeqNr, timestamp: Instant)
+
+  private case class Const[S](snapshot: S, metadata: Snapshot.Metadata)
```

Comment:
please keep things local / private with regards to scope

Reply:
fixed